Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1315,56 +1315,100 @@ static WorldObject CreateObjectForCommand(Session session, string weenieClassDes
 
         // create wclassid (number)
         [CommandHandler("create", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 1,
-            "Creates an object in the world.", "wclassid (string or number)")]
+            "Creates an object or objects in the world.", "wclassid (string or number), Amount to Spawn (default:1)")]
```

Comment:
does the description include all of the parameters?
numToSpawn / palette / stackSize?

Reply:
numToSpawn essentially equals stacksize here... palette and shade are left out.. not sure what they do really and they seem to be rarely used